src/fides/api/models/privacy_request.py

```diff
@@ -1743,6 +1743,16 @@ class TraversalDetails(FidesSchema):
     input_keys: List[str]
 
 
+    @classmethod
+    def create_empty_traversal(cls, connection_key: str):
```

Comment:
because the custom_request_fields datasets aren't necessarily reachable, they don't have a traversal node, so we need this "empty traversal" as a placeholder in some places